data/chameleoncloud/sites/uc/clusters/chameleon/nodes/048c2acc-cc3f-422e-88e2-ed1ffb48e4c9.json

```diff
@@ -5,90 +5,93 @@
     "smt_size": 96
   },
   "bios": {
-    "release_date": "02/24/2021",
-    "vendor": "Dell Inc.",
-    "version": "2.10.2"
+    "release_date": "2024-02-19",
+    "vendor": "Dell",
+    "version": "2.21.2"
   },
   "chassis": {
-    "manufacturer": "Dell Inc.",
+    "manufacturer": "Dell",
     "name": "PowerEdge R740",
     "serial": "76H6KD3"
   },
+  "gpu": {
+    "gpu": false
+  },
   "main_memory": {
     "humanized_ram_size": "192 GiB",
-    "ram_size": 192000000000
+    "ram_size": 206158430208
   },
   "monitoring": {
     "wattmeter": false
   },
   "network_adapters": [
     {
-      "device": "NIC.Integrated.1-1",
+      "device": "eno1np0",
+      "driver": "mlx5_core",
       "enabled": true,
       "interface": "Ethernet",
       "mac": "b8:ce:f6:43:4f:36",
-      "management": false,
-      "model": "MLNX 25GbE 2P ConnectX4LX RNDC",
-      "vendor": "Mellanox Technologies"
+      "model": "MT27710 Family [ConnectX-4 Lx]",
+      "rate": 25000000000,
+      "vendor": "Mellanox"
     },
     {
-      "device": "NIC.Integrated.1-2",
+      "device": "eno2np1",
+      "driver": "mlx5_core",
       "enabled": true,
       "interface": "Ethernet",
       "mac": "b8:ce:f6:43:4f:37",
-      "management": false,
-      "model": "MLNX 25GbE 2P ConnectX4LX RNDC",
+      "model": "MT27710 Family [ConnectX-4 Lx]",
       "rate": 25000000000,
-      "vendor": "Mellanox Technologies"
+      "vendor": "Mellanox"
     },
     {
-      "device": "NIC.Slot.7-1",
+      "device": "ens7f0",
+      "driver": "tg3",
       "enabled": false,
       "interface": "Ethernet",
       "mac": "e4:3d:1a:2d:d8:b6",
-      "management": false,
-      "model": "BRCM GbE 2P 5720-t Adapter",
-      "vendor": "Broadcom Inc. and subsidiaries"
+      "model": "NetXtreme BCM5720 Gigabit Ethernet PCIe",
+      "rate": 1000000000,
+      "vendor": "Broadcom"
     },
     {
-      "device": "NIC.Slot.7-2",
+      "device": "ens7f1",
+      "driver": "tg3",
       "enabled": false,
       "interface": "Ethernet",
       "mac": "e4:3d:1a:2d:d8:b7",
-      "management": false,
-      "model": "BRCM GbE 2P 5720-t Adapter",
-      "vendor": "Broadcom Inc. and subsidiaries"
+      "model": "NetXtreme BCM5720 Gigabit Ethernet PCIe",
+      "rate": 1000000000,
+      "vendor": "Broadcom"
     }
   ],
   "node_name": "P3-CPU-022",
   "node_type": "compute_cascadelake_r",
   "placement": {
     "node": "10",
-    "rack": "BG-41"
+    "rack": "5"
```

Comment:
Just a note: This data is currently being pulled from blazar (by the transmogrifier), ideally it would be pulled from Doni which would populate both blazar and the reference repo.

Calling out because "BG-41" is correct, but I think we haven't updated doni and/or blazar with that.